packages/uui-color-picker/lib/uui-color-picker.element.ts

```diff
@@ -64,10 +64,11 @@ type UUIColorPickerSize = 'small' | 'medium' | 'large';
  */
 @defineElement('uui-color-picker')
 export class UUIColorPickerElement extends LabelMixin('label', LitElement) {
-  @query('[part="input"]') _input!: UUIInputElement;
-  @query('.color-picker__preview') _previewButton!: HTMLButtonElement;
-  @query('#swatches') _swatches!: UUIColorSwatchesElement;
+  @query('[part="input"]') private _input!: UUIInputElement;
+  @query('.color-picker__preview') private _previewButton!: HTMLButtonElement;
+  @query('#swatches') private _swatches!: UUIColorSwatchesElement;
 
+  @state() private _value: string = '';
```

Comment:
I think this doesn't need to be a `@state()` since you don't seem to be syncing any rendered elements with the value...?

Maybe the `inputValue` could lose the `@state()` as well...

Reply:
`uui-color-area` seems to be rendered using this.value. Not entirely sure if this matters for _value? Perhaps skip the state and do a requestUpdate instead.... Not sure if it makes a big difference performance wise..

Reply:
If it's using `this.value`, which is a property, then it is already reactive.